src/ol-ext/wfsdescribefeaturetype.js

```diff
@@ -69,24 +80,77 @@ ol.format.WFSDescribeFeatureType.readElement_ = function(node, objectStack) {
 };
 
 
+/**
+ * @private
+ * @param {Node} node Node.
+ * @param {Array.<*>} objectStack Object stack.
+ * @return {T} Object.
+ * @template T
+ */
 ol.format.WFSDescribeFeatureType.readComplexType_ = function(node, objectStack) {
   const name = node.getAttribute('name');
-  const object = ol.xml.pushParseAndPop({'name': name}, ol.format.WFSDescribeFeatureType.COMPLEX_TYPE_PARSERS_, node, objectStack);
+  const object = ol.xml.pushParseAndPop(
+    {'name': name},
+    ol.format.WFSDescribeFeatureType.COMPLEX_TYPE_PARSERS_,
+    node, objectStack
+  );
   // flatten
-  object['complexContent'] = object['complexContent']['extension']['sequence']['element'];
+  object['complexContent'] =
+    object['complexContent']['extension']['sequence']['element'];
   return object;
 };
 
-ol.format.WFSDescribeFeatureType.readComplexContent_ = function(node, objectStack) {
-  return ol.xml.pushParseAndPop({}, ol.format.WFSDescribeFeatureType.COMPLEX_CONTENT_PARSERS_, node, objectStack);
+
+/**
+ * @private
+ * @param {Node} node Node.
+ * @param {Array.<*>} objectStack Object stack.
+ * @return {T} Object.
+ * @template T
```

Comment:
I do not know how to fix this.  This was added as an attempt to fix the issues that were thrown by the compiler.  I took what I would find in the OpenLayers other formats.  If it is not correct, please provide how to fix it.